client_http.hpp

```diff
@@ -30,6 +76,8 @@ class case_insensitive_hash {
     return seed;
   }
 };
```

Comment:
I've looked a bit on this, and could do this work I think. Never mind the above comment then:)